opta/pre_check.py

```diff
@@ -0,0 +1,23 @@
+import os
+from typing import Tuple
+
+from opta.exceptions import UserErrors
+from opta.nice_subprocess import nice_run
+
+
+def pre_check() -> None:
```

Comment:
We shouldn't run this on every command. Let's do it just on apply and deploy.

Reply:
Removed it from the main CLI, and added it to apply and destroy commands only.